.env.example

```
@@ -9,3 +9,10 @@ PLANETSCALE_PASSWORD=password
 # Cloudflare Hyperdrive
 WRANGLER_HYPERDRIVE_LOCAL_CONNECTION_STRING_POSTGRESQL=postgresql://test:test@localhost:5432/db0
 WRANGLER_HYPERDRIVE_LOCAL_CONNECTION_STRING_MYSQL=mysql://test:test@localhost:3306/db0
+
+# MSSQL
+MSSQL_HOST=localhost
+MSSQL_DB_NAME=TestDB
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MSSQL_DB_NAME key should go before the MSSQL_HOST key

(UnorderedKey)